web/src/components/Storyblok/StoryblokRichTextTable/StoryblokRichTextTable.tsx

```diff
@@ -41,6 +42,12 @@ export const StoryblokRichTextTable: React.FC<StoryblokRichTextTableProps> = ({
 	const { isFirstRowHeader, isFirstColumnHeader } = getTableHeaderConfig();
 	const headerCells = isFirstRowHeader ? firstRow.content : [];
 	const bodyRows = isFirstRowHeader ? rows.slice(1) : rows;
+	const isSummaryHidden =
+		hideSummaryText === true || hideSummaryText === "true";
+
+	const summaryTextClassName = isSummaryHidden
+		? `${styles["table__summary--visually-hidden"]}`
```

Comment:
There's a `visually-hidden` utility class we can use here rather than creating a component-specific class